tests/core/fetchIntroSteps.test.js

```diff
@@ -80,11 +81,44 @@ describe("fetchIntroSteps", () => {
     expect(steps[1].intro).toBe("second");
     expect(steps[1].step).toBe(2);
 
-    expect(steps[2].position).toBe("floating");
+    expect(steps[2].position).toBe("bottom");
     expect(steps[2].intro).toBe("third");
     expect(steps[2].step).toBe(3);
   });
 
+  test("should throw an error on calling step.element if it is not exists yet and return element after adding", () => {
+    const targetElement = document.createElement("div");
+    const elId = "later_added";
+    const steps = fetchIntroSteps.call(
+      {
+        _options: {
+          tooltipPosition: "bottom",
+          steps: [
+            {
+              element: "#" + elId,
+            },
+          ],
+        },
+      },
+      targetElement
+    );
+
+    expect(steps.length).toBe(1);
+
+    try {
+      const element = steps[0].element; // jshint ignore:line
+    } catch (e) {
+      if (!e.message.includes("There is no element with given selector:"))
+        throw e;
+    }
```

Comment:
Are we checking here to make sure the steps array is empty?

Reply:
No, we calling here `.element` of the first array item to check that Error "There is not element..." is throwing in case of element absence.